File: fibo.go
package main

import (
	"flag"
	"fmt"
	"os"
)

func fibo(x, y, z int) {
	fibonacci := []int{y, z}
	fmt.Printf("1: %d\n2: %d\n", fibonacci[0], fibonacci[1])
	for i := 2; i < x; i++ {
		fibonacci = append(fibonacci, fibonacci[i-1]+fibonacci[i-2])
		fmt.Printf("%d: %d\n", i+1, fibonacci[i])
	}
}

func main() {
	var help bool
	flag.BoolVar(&help, "h", false, "Help with using program")
	flag.Parse()
	first := flag.Int("first", 0, "First number of Fibonacci seqence")
	secound := flag.Int("secound", 1, "Secound number of Fibonacci seqence")
	num := flag.Int("num", 10, "How many numbers to calculate")
	if help == true {
		fmt.Println("You have to provide 3 arguments - first number, secound number, how many numbers to calculate\nExample:\t./fibo first=0 secound=1 num=10\n")
		os.Exit(1)
	}
	fibo(*num, *first, *secound)
	os.Exit(0)
}

File: fibo.py
#!/usr/bin/env python

from sys import argv

def fibo(x, y, z):
    fibo = [y, z]
    print(f"1: {fibo[0]}\n2: {fibo[1]}")
    for i in range(2, x):
        fibo.append(fibo[i-1] + fibo[i-2])
        print(f"{i+1}: {fibo[i]}")

if __name__ == "__main__":
    try:
        fibo(int(argv[3]), int(argv[1]), int(argv[2]))
    except:
        print("You have to provide 3 arguments - first number, secound number, how many numbers to calculate.\nExample:\t./fibo 0 1 10\n")
        yield ValueError("Not enough arguments!")